.eslintrc

```diff
@@ -55,14 +55,35 @@
     "jsx-a11y/mouse-events-have-key-events": "off",
     "jsx-a11y/click-events-have-key-events": "off",
     "jsx-a11y/no-noninteractive-element-interactions": "off",
-    "jsx-a11y/no-noninteractive-element-to-interactive-role": "off"
+    "jsx-a11y/no-noninteractive-element-to-interactive-role": "off",
+    "@typescript-eslint/camelcase": [
+      "error",
+      {
+        "allow": [
+          "UNSTABLE_Color",
+          "UNSTABLE_colors",
+          "UNSTABLE_cssCustomProperties"
+        ]
+      }
+    ]
   },
   "overrides": [
+    {
+      "files": ["src/**/*.{ts,tsx}"],
```

Comment:
Because some of our ts files aren't included in our tsconfig's files list we can't set this stuff at the global level, which is a little annoying but not the end of the world.